src/main/kotlin/no/nav/syfo/service/BehandlingService.kt

```diff
@@ -92,7 +92,7 @@ class BehandlingService constructor(
 
                 log.info("Finished processing took {}s, {}", StructuredArguments.keyValue("latency", currentRequestLatency), fields(loggingMeta))
             } else {
-                log.info("Mottatt jp som ikke treffer filter, tema = ${journalfoeringEvent.temaNytt}, mottakskanal = ${journalfoeringEvent.mottaksKanal}, hendelsestype = ${journalfoeringEvent.hendelsesType}, journalpostid = $journalpostId")
+                log.debug("Mottatt jp som ikke treffer filter, tema = ${journalfoeringEvent.temaNytt}, mottakskanal = ${journalfoeringEvent.mottaksKanal}, hendelsestype = ${journalfoeringEvent.hendelsesType}, journalpostid = $journalpostId")
```

Comment:
Da vil den aldri logge noenting med mindre den faktisk mottar noe den skal behandle..? Jeg er enig i at vi kan justere ned etterhvert, men jeg synes det er for tidlig å gjøre det nå. Jeg trenger iallefall å se at appen gjør noe :)